whelktool/scripts/cleanups/2021/04/lxl-1931-merge-series-membership.groovy

```diff
@@ -56,8 +56,8 @@ selectBySqlWhere(where) { data ->
         Map a = cmpData[it[0]]
         Map b = cmpData[it[1]]
 
-        Map mergedTrees = merge(a.tree, b.tree)
-        Map leaves = a.leavesForCmp + b.leavesForCmp
+        Map mergedLeaves = mergeLeaves(a.leavesForCmp, b.leavesForCmp, b.tree.inSeries?.instanceOf.asBoolean())
```

Comment:
```suggestion
        Map mergedLeaves = mergeLeaves(a.leavesForCmp, b.leavesForCmp, b.tree.inSeries?.instanceOf)
```
inSeries == null is already coerced to false
asBoolean() will lead to nullpointer if instanceOf is missing?
?

Reply:
@olovy actually null.asBoolean() evaluates to false so should be fine!

Reply:
Ok! TIL LOL